infrastructure/router/router.go

```diff
@@ -37,5 +37,32 @@ func NewRouter(muxRouter *mux.Router, c controllers.AppController) *mux.Router {
 		}
 	})
 
+	muxRouter.HandleFunc("/cocurrent-task", func(rw http.ResponseWriter, r *http.Request) {
+		objectsType := r.URL.Query().Get("type")
+		items := r.URL.Query().Get("items")
+		itemsPerWorker := r.URL.Query().Get("items_per_workers")
+
+		itemsInt, err := strconv.Atoi(items)
+		if err != nil {
+			rw.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		itemsPerWorkerInt, err := strconv.Atoi(itemsPerWorker)
+		if err != nil {
+			rw.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		if objectsType != "odd" && objectsType != "even" {
+			rw.WriteHeader(http.StatusBadRequest)
+			return
+		}
```

Comment:
You should also provide a message as to why this is a bad request.

Reply:
You're right, it would give more hints to the users
I will add it to the list of things to do :D